_layouts/parse.html

```diff
@@ -0,0 +1,9 @@
+{%- if site.remove_HTML_comments -%}
+	{%- assign _comments = "<!-- -->" | split: " " -%}
+	{%- capture _comment_befores -%}.{{ content }}{%- endcapture -%}
```

Comment:
is the `.` before `{{ content }}` intentional?

Reply:
It's a mistake. Solved in PR: https://github.com/alshedivat/al-folio/pull/481/commits/3cb3ebe13f6e5d89f92b21ab7d7b99a21c89e635